src/table.ts

```diff
@@ -778,133 +749,46 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       rowStream?.removeListener('end', originalEnd);
     };
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    userStream.end = (chunk?: any, encoding?: any, cb?: () => void) => {
-      rowStreamUnpipe(rowStream, userStream);
-      userCanceled = true;
-      if (activeRequestStream) {
-        activeRequestStream.abort();
-      }
-      if (retryTimer) {
-        clearTimeout(retryTimer);
-      }
-      return originalEnd(chunk, encoding, cb);
-    };
```

Comment:
This has just been moved further down. The `if (retryTimer) {` statement has been removed, but that is because `retryTimer` isn't used in this part of the code anymore.